README.md

```diff
@@ -107,27 +112,13 @@ To run all queries, cd to `tpcx_bb/` and:
 python benchmark_runner.py --config_file benchmark_runner/benchmark_config.yaml
 ```
 
-By default, this will run each query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.
+By default, this will run each pure Dask query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.
```

Comment:
Confused by this, wont this run both `DASK` and `Blazing` queries `N_REPEATS` times.

Reply:
Yes, good catch. Needs an update

Reply:
Resolved by https://github.com/rapidsai/tpcx-bb/pull/125/commits/203277c7fc94aa6ca4073b68e02794542db920db